### PR TITLE
LOG-599: Update fluent config to use static index

### PR DESCRIPTION
**Pull request by jcantrill**
This PR:
* resolves https://issues.redhat.com/browse/LOG-599
* updates the fluent config to write to a static index
* blocked by https://github.com/openshift/origin-aggregated-logging/pull/1797 and equivalent into feature branch

### CODE REVIEW

File: pkg/generators/forwarding/fluentd/templates.go [hunk 1]
```diff
@@ -235,17 +235,20 @@ const fluentConfTemplate = `{{- define "fluentConf" }}
...
-			name_type operations_full
+			name_type static
+			static_index_name infra-write
 		</elasticsearch_index_name>
 		<elasticsearch_index_name>
 			enabled "#{ENV['ENABLE_ES_INDEX_NAME'] || 'true'}"
 			tag "linux-audit.log** k8s-audit.log** openshift-audit.log**"
-			name_type audit_full
+			name_type static
+			static_index_name audit.infra-write
```
Comment (ewolinetz) [inline pkg/generators/forwarding/fluentd/templates.go]: is there a fluentd change that needs to go along with this?

Comment (jcantrill) [reply pkg/generators/forwarding/fluentd/templates.go]: yes. ref'd in the original comment: https://github.com/openshift/origin-aggregated-logging/pull/1797

**Comment by jcantrill** [comment]
blocked by #373

File: test/helpers/elasticsearch.go [hunk 2]
```diff
@@ -21,9 +21,9 @@ import (
 )
 
 const (
-	InfraIndexPrefix          = ".operations."
-	ProjectIndexPrefix        = "project."
-	AuditIndexPrefix          = ".audit."
+	InfraIndexPrefix          = "infra-"
```
Comment (lukas-vlcek) [inline test/helpers/elasticsearch.go]: @jcantrill Are we going to need to update index patterns here?

https://github.com/openshift/origin-aggregated-logging/blob/feature-es6x/elasticsearch/index_templates/com.redhat.viaq-openshift-operations.template.json#L5

Comment (jcantrill) [reply test/helpers/elasticsearch.go]: We actually neeed to make this a dash for audit https://github.com/openshift/origin-aggregated-logging/blob/feature-es6x/elasticsearch/index_templates/com.redhat.viaq-openshift-operations.template.json#L7

Comment (lukas-vlcek) [reply test/helpers/elasticsearch.go]: @jcantrill Just for the `audit.infra-*` (-> `audit-infra-*`) case or other index patterns as well?

**Pull request merged**